test/album.js

```diff
@@ -7,7 +7,7 @@ const chai = require('chai'),
   config = require('../config'),
   User = require('../app/models').User,
   { createUser, login, userOne, adminUser } = require('./util/users'),
-  { queryAlbums } = require('./util/albums'),
+  { queryAlbums, mutationAlbum, otherMutationAlbum } = require('./util/albums'),
```

Comment:
Instead of create several `mutationAlbum` with tricky names you can create a function that return a mutationAlbum with an id passed by parameters